### PR TITLE
add task related metrics

### PR DESCRIPTION
**Pull request by mqliang**
This PR add task related metrics docs

### CODE REVIEW

File: basics/components/minion.md [hunk 1]
```diff
@@ -220,3 +220,18 @@ After annotating the classes, put them under the package of name `org.apache.pin
...
 
+## Task related metrics
+
+There is a controller job runs every 5 minutes by default and emits metrics about Minion tasks scheduled in Pinot.
+For now, the following metrics are emitted for each task type:
+* ***NumMinionTasksInProgress***: Number of running tasks
+* ***NumMinionSubtasksRunning***: Number of running sub-tasks
+* ***NumMinionSubtasksWaiting***: Number of waiting sub-tasks (unassigned to a minion as yet)
+* ***NumMinionSubtasksError***: Number of error sub-tasks (completed with an error/exception)
+* ***PercentMinionSubtasksInQueue***: Percent of sub-tasks in error
```
Comment (jackjlli) [inline basics/components/minion.md:231]: The description doesn't match with the meaning of the metric here.

File: basics/components/minion.md [hunk 2]
```diff
@@ -220,3 +220,18 @@ After annotating the classes, put them under the package of name `org.apache.pin
...
+* ***PercentMinionSubtasksInError***: Percent of sub-tasks in waiting or running states
```
Comment (jackjlli) [inline basics/components/minion.md:232]: Same here.

**Review by jackjlli — CHANGES_REQUESTED** [review]

File: basics/components/minion.md (hunk 1 above)
Comment (mqliang) [reply basics/components/minion.md]: fixed

File: basics/components/minion.md (hunk 2 above)
Comment (mqliang) [reply basics/components/minion.md]: fixed

**Review by mayankshriv — APPROVED** [review]

**Pull request merged**